### DISCUSSION TITLE
Create a SWEET website

### DISCUSSION THREAD

**Discussion by rrovetto** [ESIPFed/sweet#285] [Ideas]
Proposal: create a SWEET website with a professional-looking visual landing page and format. Desired features include: eye-catching image/logo, user-friendly for all audiences, links to Github and misc. pages.

We discussed this in today's COR meeting after Slack chat. Some notes in the meeting minutes.

**Comment by semantic-nomad** [comment]
I created a quick test repo and site using gh-pages and [mkdocs](https://www.mkdocs.org/)
Repo: https://github.com/ESIPFed/sweet-website
Site: https://esipfed.github.io/sweet-website  (test URL; can easily be redirected to sweetontology.net)

Currently just the main repo's README file as content.  If this is something worth pursuing, there's probably several wiki pages that sh/could be moved.
  
The website doesn’t need to be a separate repo, but if the website files are also in main  it can get cluttered.

**Comment by semantic-nomad** [comment]
I'd like to park this as a discussion and bring it up at the next STC meeting.  In order for this, or something like it, to proceed there would need to be coordination with @esip on DNS configuration (at a minimum).

We should also consider organizing the content for that type of platform.
